### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by arya-tama** [NetEaseAI-CVLab/CNN-MonoFusion#8]
Hi,

can you provide training code for the networks?

**Comment by JackHenry1992** [comment]
Sorry for later reply, you can refer to FCRN for training. Just change the loss function and model-archi can get the results.
